build/api-gen/server-side.jsx

```diff
@@ -0,0 +1,46 @@
+React = require("react");
+
+docComp = require("./doc-components");
+var ToC = docComp.ToC;
+var DocPage = docComp.DocPage;
+var EventPage = docComp.EventPage;
+
+
+// We create a StaticPage component for generating the page
+// Unlike in thre dynamic case, we assume we can just pass the necessary state directly
+var StaticPage = React.createClass({
+	render: function() {
+		var pages = this.props.index.pages;
```

Comment:
Using tabs here? :)

Should we point grunt at validating files in `build` as well? Maybe editorconfig? http://editorconfig.org/